packages/dds/matrix/src/test/utils.ts

```diff
@@ -222,3 +224,33 @@ export function insertFragmented(
 
 	return matrix;
 }
+
+/**
+ * Creates a local matrix with the specified size and for dense test matrix given initial value.
+ * Otherwise, leaving the initial value as undefined will create a sparse matrix.
+ * @param id - The id of the matrix.
+ * @param size - The size of the matrix.
+ * @param initialValue - The initial value of each cell in the dense matrix.
+ */
+export function createLocalMatrix({
+	id,
+	size,
+	initialValue,
+}: {
+	id: string;
+	size: number;
+	initialValue?: string;
```

Comment:
Nit: a couple pieces of feedback here:
- When using property bag input, tooling prefers that you inline `@param` comments onto the individual properties.
- I would also recommend making the properties `readonly` (generally just a best practice, unless something needs to be mutable).
- We should document clearly that "size" refers to the number of rows _and_ columns.
- We should note that whether or not `initialValue` is provided determines whether the matrix will be dense or sparse.

```suggestion
 */
export function createLocalMatrix({
	id,
	size,
	initialValue,
}: {
	/** The id of the matrix. */
	readonly id: string;
	/** The number of rows and columns that will be in the matrix. */
	readonly size: number;
	/** The initial value of each cell in the dense matrix. If not specified, no cell values will be inserted into the table, leaving it sparse. */
	readonly initialValue?: string;
```

Reply:
Applied. Changed to // since `error  Single line blocks are not permitted by your configuration  jsdoc/multiline-blocks`